torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -3235,6 +3238,28 @@ def test_ddp_logging_data_cpu(self):
             self.assertEqual(ddp_logging_data.find_unused_parameters, False)
             self.assertEqual(ddp_logging_data.gradient_as_bucket_view, False)
             self.assertEqual(ddp_logging_data.backend_name, dist.get_backend(group_id))
+            self.assertEqual(ddp_logging_data.iteration, 0)
+            params = list(model_DDP.parameters())
+            num_params = 0
+            param_size = 0
+            params = list(parameter for parameter in filter(lambda parameter: parameter.requires_grad, params))
+            for p in params:
+                num_params += 1
+                param_size += p.numel() * p.element_size()
+            self.assertEqual(ddp_logging_data.dtype, "float")
+            self.assertEqual(ddp_logging_data.parameter_size, param_size)
+            self.assertEqual(ddp_logging_data.num_parameters, num_params)
+            self.assertEqual(ddp_logging_data.bucket_sizes, [param_size])
```

Comment:
Should we have a test where there is more than one bucket?

Reply:
added